src/Widgets/Helper/Factories/Settings/ValueCaptionFactory.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+
+namespace Ceres\Widgets\Helper\Factories\Settings;
+
+
+class ValueCaptionFactory
+{
+    private $valueList;
+
+    public function addEntry($key, $value)
+    {
+        $this->valueList[] = [$key => $value];
```

Comment:
```suggestion
        $this->valueList[] = [
            "value" => $value,
            "caption" => $key
        ];
```

ich würde evtl. $key dann in $caption umbenennen, um klar zu machen, dass es sich um einen Lesbaren Wert handelt, der in der UI dargestellt wird und nicht um einen Key, der codeseitig Verwendung findet.